server/resources/run.py

```diff
@@ -74,6 +54,7 @@ def rename(self, rfolder: dict, name: str) -> dict:
         return super().rename(rfolder, name)
 
     @access.user(TokenScope.DATA_READ)
+    @filtermodel('folder')
```

Comment:
For the recorded run task, I need the `runVersionId` which is filtered out with this, I think.

Reply:
Fixed in https://github.com/whole-tale/wt_versioning/pull/30/commits/1386be8638d0bc5fab1d0f130462f7d588427054